lib/register.coffee

```diff
@@ -19,7 +26,12 @@ exports.generateUUID = ->
 	# pass the certificate validation in OpenVPN This either means that
 	# the RFC counts a final NULL byte as part of the CN or that the
 	# OpenVPN/OpenSSL implementation has a bug.
-	return crypto.pseudoRandomBytes(31).toString('hex')
+	Promise.try ->
+		crypto.randomBytes(31).toString('hex')
+	.catch ->
```

Comment:
@jviotti added a test with this mock, and addressed all the other comments. What do you think?